package.json

```diff
@@ -14,6 +14,10 @@
     "build": "shx rm -rf dest && npm run fetch && run-p build:images build:js build:pug build:sass",
     "build-uncompressed": "shx rm -rf dest && npm run fetch && run-p build:images build:js-uncompressed build:pug build:sass",
     "fetch": "node scripts/fetch-json.js",
+    "optimize:svg": "find source/images -type f -name '*.svg' -print0 | xargs -0 -n 1 -P 6 svgo --multipass --pretty --enable=removeTitle",
```

Comment:
Yeah, we can file a follow-on to refactor as a Node script, but as-is this will only be run maybe once a month or so by Alan or myself.